node/flatpak-node-generator.py

```diff
@@ -67,6 +67,68 @@
 NPM_MIRROR = 'https://unpkg.com/'
 
 
+class SemVer(NamedTuple):
```

Comment:
Why create a custom semver parser instead of using the `semver` module? Doesn't it work for npm package versions?